src/stackwalk.c

```diff
@@ -517,27 +547,19 @@ static int jl_unw_step(bt_cursor_t *cursor, uintptr_t *ip, uintptr_t *sp, uintpt
     if (unw_get_reg(cursor, UNW_REG_SP, &reg) < 0)
         return 0;
     *sp = reg;
-#ifdef UNW_REG_FP
-    if (unw_get_reg(cursor, UNW_REG_FP, &reg) < 0)
-        return 0;
-    if (fp)
-        *fp = reg;
-#else
-    if (fp)
-        *fp = 0;
-#endif
+    *fp = 0;
```

Comment:
So it looks like we don't need `fp` at all anymore? In that case would it be better just to remove this parameter from `jl_unw_step`?